pkgs/applications/science/astronomy/gildas/default.nix

```diff
@@ -0,0 +1,58 @@
+{ stdenv, fetchurl, gtk2 , pkgconfig , python27 , gfortran , python27Packages , lesstif , cfitsio , getopt , perl , groff }:
+
+stdenv.mkDerivation rec {
+  srcVersion = "jul17a";
+  version = "20170701_a";
+  name = "gildas-${version}";
+
+  src = fetchurl {
+    url = "http://www.iram.fr/~gildas/dist/gildas-src-${srcVersion}.tar.gz";
+    sha256 = "060vha4af2xamk0fvriqr6rgq4zayy4k6l5y6kj3h5cv9b4yiqzq";
+  };
+
+  enableParallelBuilding = false;
+
+  nativeBuildInputs = [ pkgconfig groff perl getopt gfortran python27 python27Packages.numpy ];
```

Comment:
Is `numpy`  a runtime dependency of this application?

Reply:
> Is `numpy` a runtime dependency of this application?

Yes, it is.